### PR TITLE
Fix building of rackspace-ccm

### PR DESCRIPTION
**Pull request by pshanbhag**
Still requires a amd64 system to build though

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,7 +30,7 @@ clean:
 	rm -rf $(TARGET)
 
 image: build
-	docker build -t $(REGISTRY)/$(TARGET):latest .
-
+	docker build -t $(REGISTRY)/$(TARGET):latest . && \
+        docker tag $(REGISTRY)/$(TARGET):latest $(REGISTRY)/$(TARGET):${VERSION}
```
Comment (supreeth90) [inline Makefile:34]: why not build directly with `${VERSION}` tag. ?

File: go.mod [hunk 2]
```diff
@@ -122,7 +122,7 @@ require (
 )
 
 replace (
-	github.com/gophercloud/gophercloud => /home/xagent/gophercloud
-	github.com/gophercloud/utils => /home/xagent/utils
-	k8s.io/cloud-provider => /home/xagent/k8s-cloud-provider
+	github.com/gophercloud/gophercloud => ../gophercloud
```
Comment (supreeth90) [inline go.mod:125]: should we tie them to a version ?

Comment (supreeth90) [reply go.mod:125]: we dont have to worry about checking it out seperately every time we try to build them, unless we have some custom changes in gophercloud.

Comment (pshanbhag) [reply go.mod:125]: I am not sure why it is done this way. But I will come to this in a separate PR if it is not critical.

File: Makefile (hunk 1 above)
Comment (pshanbhag) [reply Makefile:34]: I assumed it is useful to have the latest tag locally. Let me get rid of it.

**Review by supreeth90 — APPROVED** [review]

**Pull request merged**